services/_shared/llm_client.py

```
@@ -1326,6 +1327,199 @@ def review_diff(
+        # so a bad config can never raise out of review_diff.
+        cfg_start_ns = time.monotonic_ns()
+        with _llmobs_llm(
+            model_name=backend.value, model_provider=backend.value, name=_LLMOBS_NAME,
+        ) as cfg_span:
+            _llmobs_annotate(
+                span=cfg_span, input_data=_redact_payload(messages),
+                metadata={"backend": backend.value, "variant_id": variant, "error": "config"},
+                metrics={"latency_ms": _elapsed_ms(cfg_start_ns)},
+                tags=pr_tags,
+            )
+        log.error("llm_backend_misconfigured", extra={"backend": backend.value, "detail": str(e)})
```

Ruff (0.15.21)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)